docs/design.rst

```diff
@@ -97,14 +99,16 @@ concatenate broken words would need a function that removes the element objects
 in question while preserving the text fragments in its meaningful sequence
 attached to the ``text`` and ``tail`` properties. In case these have no content,
 their value of ``None`` leads to different operations to concatenate strings.
-Here's a working implementation from the inxs_ library that is used by a variety
-of more specific functions:
+Here's a working implementation from the inxs_ library for that data model:
 
 .. code-block:: python
 
-   def remove_elements(*elements: etree.ElementBase, keep_children=False,
-                       preserve_text=False,
-                       preserve_tail=False) -> None:
+   def remove_elements(
+       *elements: etree.ElementBase,
```

Comment:
```suggestion
       *nodes: delb.NodeBase,
```

Reply:
isn't this the delbless example!?

Reply:
no, no more at this point. it's about the same task with the same data type, but using a different data model.

Reply:
oh no, this sparse-context view git me again. sorry, fixing it.